CleverRaven/Cataclysm-DDA
# Spawn `liq_bandage_spray`, not `liq_bandage`

#### Summary
Bugfixes "Spawn `liq_bandage_spray`, not `liq_bandage`"

#### Purpose of change
Fix #64150

#### Describe the solution
Spawn the spray bottle and not uncontained liquid.

#### Testing
- Deleted everything else from `gym` itemgroup in `itemgroups/Locations_MapExtras/locations_commercial.json`
- Use debug to put a fitness gym `gym_fitness` on the map
- Visit the gym and check benches/lockers to verify item spawned is as intended

![Screenshot from 2023-03-13 00-24-09](https://user-images.githubusercontent.com/28502722/224558199-87c4f32b-6d64-45ac-a581-c4e4c8a1fbb5.png)
